curriculum/challenges/english/blocks/workshop-bookstore-page/69134b6cb43aa5f254950a10.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 69134b6cb43aa5f254950a10
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+
+# --description--
+
+
+In this workshop, you will build a bookstore page by creating book cards that display information about different books. You'll practice organizing content using `div` elements, classes, and IDs.
+
+Start your bookstore page by creating the HTML boilerplate.
+
+Add the `<!DOCTYPE html>` declaration and `html`, `head`, and `title` elements.
+
+Set the page title to `XYZ Bookstore Page`.
+
+# --hints--
+
+Your code should include a `<!DOCTYPE html>` declaration.
+
+```js
+assert.match(code, /<!DOCTYPE html>/i);
+```
+
+Your code should include opening and closing `<html>` tags.
+
+```js
+assert.exists(document.querySelector('html'));
+```
+
+Your code should include opening and closing `<head>` tags.
+
+```js
+assert.exists(document.querySelector('head'));
+```
+
+Your code should include opening and closing `<title>` tags. 
+
+```js
+assert.exists(document.querySelector('title'));
```

Comment:
I would check that the tile is inside the `head`. Probably the hint should be updated as well.
```suggestion
assert.exists(document.querySelector('head > title'));
```